### PR TITLE
feat: Unit testing for user id priority

### PR DESCRIPTION
**Pull request by JamieSinn**

### CODE REVIEW

File: .github/workflows/run-test-harness.yml [hunk 1]
```diff
@@ -14,5 +14,3 @@ jobs:
         with:
           sdks-to-test: php
           sdk-github-sha: ${{github.event.pull_request.head.sha}}
-          github-token: ${{ secrets.TEST_HARNESS_GH_SECRET }}
```
Comment (jonathannorris) [inline .github/workflows/run-test-harness.yml:17]: why is this removed?

**Review by jonathannorris — APPROVED** [review]

File: .github/workflows/run-test-harness.yml (hunk 1 above)
Comment (JamieSinn) [reply .github/workflows/run-test-harness.yml:17]: not used anymore - we have it using the built in auth.
